Composer/packages/client/src/ShellApi.ts

```diff
@@ -1,13 +1,12 @@
+/*eslint @typescript-eslint/no-use-before-define: ["error", { "functions": false }]*/
```

Comment:
do we really need to change this eslint rule here? Can we avoid this?

Reply:
resolved